src/errors.ts

```diff
@@ -112,3 +116,43 @@ export enum ValidationErrorCodes {
     InternalError = 1008,
     TokenNotSupported = 1009,
 }
+
+export abstract class AlertError {
+    public abstract message: string;
+    public shouldAlert: boolean = true;
```

Comment:
I don't see this public boolean used anywhere, is that a `TODO` ?

Reply:
it's not used anywhere, but I was envisioning it being useful when searching logs.

E.g. we could search for `parsed_log.shouldAlert = true` and use that as a Grafana alert rule.

Reply:
Ah I see! So basically a flag that resembles a higher log level